src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -91,4 +91,15 @@ const DetailsSidebar = ({
     </SidebarContent>
 );
 
+export type DetailsSidebarProps = {
```

Comment:
Curious as to the reasoning of the different prop type, considering the substantial overlap with the above `Props` type.

Reply:
The above type is everything the details tab expects. This type a way to document what we expect to be passed into the sidebar by the consuming app. For flow purposes, this needs to be a type so it can be properly typed in `ContentSidebar`

Reply:
Ah, I see. Are compound types available in Flow? If so, could we have `Props` include the props that DO NOT exist in `DetailsSidebarProps`, and have it extend/include `DetailsSidebarProps`;

IE) 
```
type DetailsSidebarProps = {
  my_prop: boolean
};

type Props = {
  your_prop: string
} | DetailsSidebarProps;
```

Reply:
yeah I can try to do something like that!